parm/config/gfs/config.earc

```diff
@@ -8,7 +8,22 @@ echo "BEGIN: config.earc"
 # Get task specific resources
 . $EXPDIR/config.resources earc
 
-export NMEM_EARCGRP=10
+# Set the number of ensemble members to archive per earc job
+case "${CASE_ENS}" in
+   "C48" | "C96")
+      export NMEM_EARCGRP=80
+      ;;
+   "C192")
+      export NMEM_EARCGRP=20
+      ;;
+   "C384" | "C768")
+      export NMEM_EARCGRP=10
+      ;;
+   *)
+      echo "FATAL ERROR: Unknown ensemble resolution ${CASE_ENS}, ABORT!"
+      exit 1
+      ;;
+esac
```

Comment:
We need a C1152 case. That is already being run for HR3.

Reply:
Added an 1152 case, though I haven't tested it.  4 is a fairly conservative estimate.